vernac/vernacentries.ml

```diff
@@ -2339,8 +2339,10 @@ let translate_vernac ~atts v = let open Vernacextend in match v with
         vernac_import export qidl)
   | VernacCanonical qid ->
     VtDefault(fun () ->
-        unsupported_attributes atts;
-        vernac_canonical qid)
+        vernac_canonical
+          ~poly:(only_polymorphism atts)
+          ~scope:(DeclareDef.Global Declare.ImportDefaultBehavior)
```

Comment:
Why not get it from the attributes?

Reply:
(while `Canonical x := y.` is like Definition, this is why I've added the scope/poly arguments)

Reply:
And it seemed wrong to set `poly:false`, since `y` may be a polymorphic constant.

Reply:
> I mean why not change that, since you already did the work to pass the scope around?

OK, it could be done, but seems orthogonal to this PR (for sure it is not required). And I'm not familiar with the libobject for canonical...